.eslintrc.json

```diff
@@ -18,6 +18,7 @@
     "require-jsdoc": 2,
     "valid-jsdoc": 2,
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
-    "jsx-quotes": [2, "prefer-single"]
+    "jsx-quotes": [2, "prefer-single"],
+    "no-return-assign": [0]
```

Comment:
I suggest to use `"no-return-assign": [2, "except-parens"]` instead, since rule is quite useful. If you are sure that assignment is correct, you can wrap it in parentheses.

Reply:
K.